src/components/views/rooms/RoomUpgradeWarningBar.js

```diff
@@ -74,6 +82,33 @@ module.exports = React.createClass({
                         {_t("Upgrade this room to the recommended room version")}
                     </AccessibleButton>
                 </p>
+            </div>
+        );
+
+        if (this.state.upgraded) {
+            doUpgradeWarnings = (
+                <div className="mx_RoomUpgradeWarningBar_body">
+                    <p>
+                        {_t("This room has already been upgraded.")}
```

Comment:
For a moment, I thought it could be nice to link the new room here, but I suppose that should already be on screen in the timeline for everyone to access the new room?